app/assets/javascripts/views/restrooms/restrooms.js.coffee

```diff
@@ -3,15 +3,20 @@
 # You can use CoffeeScript in this file: http://coffeescript.org/
 
 $ ->
-  $('#ada_filter').click -> 
+  $('#ada_filter').click ->
     if $(this).hasClass("active")
       $('.listItem.not_accessible').show()
-    else 
+    else
       $('.listItem.not_accessible').hide()
-    
-  $('#unisex_filter').click -> 
+
+  $('#unisex_filter').click ->
     if $(this).hasClass("active")
       $('.listItem.not_unisex').show()
-    else 
+    else
       $('.listItem.not_unisex').hide()
 
+  $('#changing_table_filter').click ->
```

Comment:
I too don't think it's necessary at this point but I think we'd want to use a bitfield, not an enum. Here's a popular one that's named after a silly looking dog https://github.com/pboling/flag_shih_tzu :)